Sami/RemoteRepository/GitHubRemoteRepository.php

```diff
@@ -15,7 +15,7 @@ class GitHubRemoteRepository extends AbstractRemoteRepository
 {
     public function getFileUrl($projectVersion, $relativePath, $line)
     {
-        $url = 'https://github.com/'.$this->name.'/blob/'.$projectVersion.$relativePath;
+        $url = 'https://github.com/'.$this->name.'/blob/'.str_replace('\\', '/',$projectVersion.$relativePath);
```

Comment:
I copied the solution from [here](https://github.com/FriendsOfPHP/Sami/blob/master/Sami/Renderer/TwigExtension.php#L62). Should those use `DIRECTORY_SEPARATOR` as well?